### PR TITLE
[FIX] l10n_lu: add taxes for ATN

### PR DESCRIPTION
**Pull request by william-andre**
Necessary Corrections for benefit in kind (ATN 17%) & Tax Report

task-[2500543](https://www.odoo.com/web#id=2500543&model=project.task&view_type=form&cids=1&menu_id=)

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_lu/data/account_tax_report_line.xml [hunk 1]
```diff
@@ -463,6 +464,7 @@
       <field name="sequence">6</field>
       <field name="parent_id" ref="account_tax_report_line_2a_breakdown_taxable_turnover_base"/>
       <field name="country_id" ref="base.lu"/>
+      <field name="code">LUTAX_701</field>
```
Comment (oco-odoo) [inline addons/l10n_lu/data/account_tax_report_line.xml:467]: Not used?

**Review by oco-odoo — CHANGES_REQUESTED** [review]

File: addons/l10n_lu/data/account_tax_report_line.xml (hunk 1 above)
Comment (william-andre) [reply addons/l10n_lu/data/account_tax_report_line.xml]: fixed the missing `account_id`s and a `plus` instead of `minus` too

**Review by oco-odoo — APPROVED** [review]
@robodoo r+

**Comment by oco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**